### ISSUE TITLE
Update the Upgrade Kubeflow page (Google Cloud) for 1.4

### ISSUE THREAD

**Issue by shannonbradshaw** [kubeflow/website#2998]
This page explicitly mentions v1.3.

https://www.kubeflow.org/docs/distributions/gke/deploy/upgrade/

**Comment by zijianjoy** [comment]
Yes this is expected. We have refactored significantly at v1.3. So this page is just to say that we do not support any upgrade from pre-v1.3 to v1.3.

I have walked through this page and this is up-to-date.
